lib/ansible/module_utils/facts/hardware/linux.py

```diff
@@ -577,7 +582,13 @@ def get_mount_facts(self):
 
         # start threads to query each mount
         results = {}
-        pool = ThreadPool(processes=min(len(mtab_entries), cpu_count()))
+        pool = None
+        try:
+            pool = ThreadPool(processes=min(len(mtab_entries), cpu_count()))
+        except Exception as e:
```

Comment:
Can we handle `PermissionError` explicitly? Catch-alls are bad for debugging.

Reply:
It is not the only possible error, was keeping exception until i could compile a full list